File: abc/CharacterGenerator/Lambda.go
package main

import "fmt"

func main() {
    const (
        FIRST_GLYPH = 33 // '!'
        LAST_GLYPH = 126 // '~'
        GLYPHS_PER_LINE = 72
    )

    cursor := FIRST_GLYPH

    var i int
    for {
        i = cursor
        for j := 0; j < GLYPHS_PER_LINE; j++ {
            fmt.Print(string(i))
            if i == LAST_GLYPH {
                i = FIRST_GLYPH
            } else {
                i += 1
            }
        }
        fmt.Println()
        if cursor == LAST_GLYPH {
            cursor = FIRST_GLYPH
        } else {
            cursor += 1
        }
    }
}

File: abc/CharacterGenerator/Lambda.cpp
#include <iostream>

using namespace std;

int main() {
    const int FIRST_GLYPH = 33; // '!'
    const int LAST_GLYPH = 126; // '~'
    const int GLYPHS_PER_LINE = 72;

    int cursor = FIRST_GLYPH;

    int i;
    while (true) {
        i = cursor;
        for (int j = 0; j < GLYPHS_PER_LINE; j++) {
            cout << static_cast<char>(i);
            if (i == LAST_GLYPH) {
                i = FIRST_GLYPH;
            } else {
                i++;
            }
        }
        cout << endl;
        if (cursor == LAST_GLYPH) {
            cursor = FIRST_GLYPH;
        } else {
            cursor++;
        }
    }
}
